src/components/Header/header.jsx

```diff
@@ -56,8 +93,8 @@ export class Header extends Component {
         isArrowDisplayed
       >
         <Menu.Item text="Add CDS Services (placeholder)" key="add-services" onClick={this.addCDSServices} />
-        <Menu.Item text="Change Patient (placeholder)" key="change-patient" />
-        <Menu.Item text="Change FHIR Server (placeholder)" key="change-fhir-server" />
+        <Menu.Item text="Change Patient" key="change-patient" onClick={() => this.openChangePatient()} />
```

Comment:
No, it should have been an event call back, but tried to ignore the event object. I was trying to get around the callback function automatically being passed in the event object, since the callback accepts one parameter already. Fixed this so that we add the event object being passed in, and ignore it. It's more transparent this way I think.